mypy/strconv.py

```diff
@@ -24,10 +24,9 @@ class StrConv(NodeVisitor[str]):
 
     def __init__(self, show_ids: bool = False) -> None:
         self.show_ids = show_ids
+        self.id_mapper = None  # type: IdMapper
```

Comment:
Why not Optional? (Currently mypy ignores the None initialization but we may change that, and it seems to me this function can definitely return with `id_mapper` still None.